hazelcast/src/main/java/com/hazelcast/map/impl/operation/ClearNearCacheOperation.java

```diff
@@ -18,6 +18,7 @@
 
 import com.hazelcast.spi.impl.MutatingOperation;
 
+// TODO not used operation remove it.
```

Comment:
Yes it will not be used any more but we will keep it for client protocol backward compatibility reasons.